pkg/cli/cli.go

```diff
@@ -182,8 +188,8 @@ func (cl *CommandLineInterface) setUntouchedFlagValuesToNil() error {
 	return nil
 }
 
-// processRangeFilterFlags sets min and max to the appropriate 0 or maxInt bounds based on the 3-tuple that a user specifies for base flag, min, and/or max
-func (cl *CommandLineInterface) processRangeFilterFlags() error {
+// ProcessRangeFilterFlags sets min and max to the appropriate 0 or maxInt bounds based on the 3-tuple that a user specifies for base flag, min, and/or max
+func (cl *CommandLineInterface) ProcessRangeFilterFlags() error {
```

Comment:
why do we need to make this and SetUntouchedFlagValuesToNil public? It doesn't look like you're using them in any new locations

Reply:
the cli package can be used outside of this project to perform integrations with instance-selector and still have access to instance-selector helpers for special flags.  Instance-selector is able to call ParseAndValidate which performs all processing, but when used in another cobra app, we need the flexibility to call individual processing functions.